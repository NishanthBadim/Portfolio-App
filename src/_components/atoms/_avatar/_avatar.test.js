import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import { render, screen, getByRole, fireEvent } from "@testing-library/react";
import Avatar from './_avatar'

test("Match the snapshot - Avatar",() => {
    const input=create(<Avatar />);
    expect(input.toJSON()).toMatchSnapshot();
})

test("Image is present", () => {
    render(<Avatar/>);
    const image=screen.getByAltText("avatar");
    expect(image).toBeInTheDocument();
})