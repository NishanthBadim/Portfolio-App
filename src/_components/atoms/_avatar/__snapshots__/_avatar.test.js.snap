// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Avatar 1`] = `
<img
  alt="avatar"
  className="makeStyles-root-1"
  src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRDXh6VSPmGsJQ8SOQP5vdAXibWEJSd1ZXUZA&usqp=CAU"
/>
`;
